buildpacks/ruby/Cargo.toml

```
@@ -23,6 +23,7 @@ tempfile = "3"
 thiserror = "1"
 ureq = "2"
 url = "2"
+clap = { version = "4.4.1", features = ["derive"] }
```

The pattern we're been using (and can be seen in the lines above), is major versions only in `Cargo.toml`:

```suggestion
clap = { version = "4", features = ["derive"] }
```